engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -27,7 +27,8 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
 
             $scope.repository.$promise.then(function () {
                 $scope.uploadURL = 'katello/api/v2/repositories/' + $scope.repository.id + '/upload_content';
-                $scope.repository['ignore_srpms'] = $scope.repository['ignorable_content'].includes("srpm");
+                $scope.repository['ignore_srpms'] = $scope.repository['ignorable_content'] && $scope.repository['ignorable_content'].includes("srpm");
+                $scope.repository['ansible_collection_auth_exists'] = $scope.repository['ansible_collection_auth_url'] && $scope.repository['ansible_collection_auth_token'];
```

Comment:
If you make a function ie `ansibleCollectionAuthExists()` then there's no need to track or nullify it, or is there a reason not to do that? :thinking:

Reply:
I tried that..The function doesn't wait for `promise.then` before the 1st render ignoring what the actual repository record has..

Reply:
Ah! Ok, let's forget about that then :)